airflow/executors/base_executor.py

```diff
@@ -332,12 +351,12 @@ def trigger_tasks(self, open_slots: int) -> None:
 
         :param open_slots: Number of open slots
         """
-        span = Trace.get_current_span()
```

Comment:
Do we not want to have the tracer anymore?

Reply:
l'm also curious about the state of this one.